enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -190,7 +196,9 @@
   [path {:keys [state user v2]
          :or {state :active}
          :as opts}]
-  (mdb/setup-db!) (db/select 'User)
+  (log/tracef "Dumping to %s with options %s" (pr-str path) (pr-str opts))
+  (mdb/setup-db!)
+  (db/select User) ;; TODO -- why???
```

Comment:
I think that's an expression I entered for debugging and never deleted. Go ahead and remove it.